x-pack/plugins/apm/public/components/shared/charts/transaction_error_rate_chart/index.tsx

```diff
@@ -33,9 +42,26 @@ export function TransactionErrorRateChart({
   const { serviceName } = useParams<{ serviceName?: string }>();
   const { urlParams, uiFilters } = useUrlParams();
   const { transactionType } = useApmServiceContext();
-  const { start, end, transactionName } = urlParams;
+  const {
+    start,
+    end,
+    transactionName,
+    comparisonEnabled,
+    comparisonType,
+  } = urlParams;
+  const {
+    comparisonStart = undefined,
+    comparisonEnd = undefined,
+  } = comparisonType
```

Comment:
Isn't `undefined` redundant?

```suggestion
  const {
    comparisonStart,
    comparisonEnd,
  } = comparisonType
```